docs/source/en/model_doc/led.mdx

```diff
@@ -44,8 +44,10 @@ Tips:
 - LED makes use of *global attention* by means of the `global_attention_mask` (see
   [`LongformerModel`]). For summarization, it is advised to put *global attention* only on the first
   `<s>` token. For question answering, it is advised to put *global attention* on all tokens of the question.
-- To fine-tune LED on all 16384, it is necessary to enable *gradient checkpointing* by executing
-  `model.gradient_checkpointing_enable()`.
+- To fine-tune LED on all 16384, *gradient checkpointing* can be enabled in case training leads to out-of-memory (OOM)
+  errors. This can be done by either executing `model.gradient_checkpointing_enable()` or using the 
+  `gradient_checkpointing=True` flag in `.from_pretrained(...)` when loading the model. Moreover, the `use_cache=False`
```

Comment:
I am not very sure about 
```
`gradient_checkpointing=True` flag in `.from_pretrained(...)` when loading the model.
```
as I see
https://github.com/huggingface/transformers/blob/b0733457a901e5ac197a8e8f183ea25d972801cd/src/transformers/configuration_utils.py#L361-L367

It looks like we would rather use `gradient_checkpointing` for `TrainingArguments`.

Reply:
Agree with @ydshieh, it should be passed to the `TrainingArguments`